resources/views/movies/edit.blade.php

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
You can also use the layout here ;D

And we do not need a view to delete a movie we can just have a button on the index that only an admin can see or something.